spec/models/manageiq/providers/amazon/storage_manager/s3/stubbed_refresher_spec.rb

```diff
@@ -4,14 +4,14 @@
 describe ManageIQ::Providers::Amazon::StorageManager::S3::Refresher do
   include AwsStubs
 
-  describe "refresh" do
-    before do
-      _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
-      @ems                 = FactoryGirl.create(:ems_amazon, :zone => zone)
-      @ems.update_authentication(:default => {:userid => "0123456789", :password => "ABCDEFGHIJKL345678efghijklmno"})
-      EvmSpecHelper.local_miq_server(:zone => Zone.seed)
-    end
+  before do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems                 = FactoryGirl.create(:ems_amazon, :zone => zone)
```

Comment:
for future code, always prefer `let` over instance vars in before blocks. The latter will lead to object instantiated even if they are not used in the specs.